src/utils/tests.jsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { Router } from 'react-router-dom';
 import { createMemoryHistory } from 'history';
 import { render } from '@testing-library/react';
+import { AppContext } from '@edx/frontend-platform/react';
 
-// eslint-disable-next-line import/prefer-default-export
+// eslint-disable import/prefer-default-export
```

Comment:
slightly unrelated, but this line makes me want to go look up what the default keyword actually does in export calls. I've never not used it and I don't even know what it does. Oops.

Reply:
I'm not exactly sure what the implementation is, but if I do:
`import Foo from './foo'` I will get whatever was exported as default. Importantly, I will get it, and it will be called **Foo**, _regardless_ of it's original name.
However, if I do `import { Foo } from './foo'`, it will go looking in the exports for something named `Foo`.
This is, incidentally why you should not export anything in your file with the same name as your file, because you could end up in a situation where you want to do 
`import Foo, {Foo as Bar} from './foo'` which, uh, works, but is confusing for everyone.

Reply:
oh yeah that is confusing. Good to know!